.pre-commit-config.yaml

```diff
@@ -30,6 +30,10 @@ repos:
         name: Run Ruff (lint) on Tools/wasm/
         args: [--exit-non-zero-on-fix, --config=Tools/wasm/.ruff.toml]
         files: ^Tools/wasm/
+      - id: ruff-check
+        name: Run Ruff (lint) on Apple/
+        args: [--exit-non-zero-on-fix, --config=Apple/.ruff.toml]
+        files: ^Apple/
```

Comment:
We (mostly) have these in alphabetical order by `files`, please could you move this to the top of the ruff-check block?

Likewise move the ruff-format up to here.